src/nat/util.rs

```diff
@@ -38,13 +38,13 @@ pub fn enable_so_reuseport(_sock: &TcpBuilder) -> io::Result<()> {
 /// A replacement for `IpAddr::is_global` while we wait for that to enter stable.
 pub fn ip_addr_is_global(ip: &IpAddr) -> bool {
     match *ip {
-        IpAddr::V4(ref addr_v4) => ipv4_addr_is_global(addr_v4),
```

Comment:
I would probably put these in a separate commit (about making latest clippy happy) like the rustfmt change, but yymv

Reply:
Done!